operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -894,6 +894,16 @@ function wdtCreatePrimordialDomain() {
 
   trace "About to call '${WDT_BINDIR}/createDomain.sh ${wdtArgs}'."
 
+  trace "DEBUG: WLSDEPLOY_PROPERTIES is ${WLSDEPLOY_PROPERTIES}"
+
+  cd ${DOMAIN_HOME} || exitOrLoop
+  for file in $(sort_files ${IMG_ARCHIVES_ROOTDIR} "*.zip")
```

Comment:
why not factor it in a method?

Reply:
ok. Originally I was thiking that it's only needed for updateDomain.sh, I'll refactor it into a method.